components/Layout/FullLayout/FullLayou.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { ILayout } from "../types"
+import Navbar from "../../Navbar"
```

Comment:
please use `absolute` imports if more than 1 `../`
```suggestion
import Navbar from "@/components/Navbar"
```

Reply:
fixed ✅